contrib/google.golang.org/grpc/grpc.go

```diff
@@ -31,11 +31,13 @@ func startSpanFromContext(ctx context.Context, method, operation, service string
 	return tracer.StartSpanFromContext(ctx, operation, opts...)
 }
 
-// withStreamError returns a tracer.WithError finish option, disregarding OK, EOF and Canceled errors.
-func withStreamError(err error) tracer.FinishOption {
+// finishWithStreamError applies finish option and a tag with gRPC status code, disregarding OK, EOF and Canceled errors.
```

Comment:
We should call this as suggested: `finishWithError`. It applies to both stream & unary now.